mirar/processors/astromatic/sextractor/sextractor.py

```diff
@@ -122,14 +126,18 @@ def __init__(  # pylint: disable=too-many-locals
         self.cache = cache
         self.mag_zp = mag_zp
         self.write_regions = write_regions_bool
-        self.psf_name = psf_name
+        self.use_psfex = use_psfex
+        self.psf_path = psf_path
         self.catalog_purifier = catalog_purifier
 
         if isinstance(self.checkimage_name, str):
             self.checkimage_name = [self.checkimage_name]
         if isinstance(self.checkimage_type, str):
             self.checkimage_type = [self.checkimage_type]
 
+        if ~self.use_psfex & (self.psf_path is not None):
```

Comment:
if (not self.use_psfex) & (self.psf_path is not None):

(I think it's more readable/pep8-y)